### DISCUSSION TITLE
WrapPanel 虚拟化问题

### DISCUSSION THREAD

**Discussion by aixiao2u1mei** [WPFDevelopersOrg/WPFDevelopers.Minimal#2] [Q&A] [answered]
Hello, will you consider the virtualization of WrapPanel later?

**Comment by yanjinhuagood** [comment, marked as answer]
> 您好，您以后会考虑WrapPanel的虚拟化吗？

请关注 https://github.com/WPFDevelopersOrg/WPFDevelopers 项目，此项目是自定义库，应该会添加到这里。

**Reply by aixiao2u1mei** [reply]
谢谢

**Comment by yanjinhuagood** [comment]
不客气

**Comment by yanjinhuagood** [comment]
> Hello, will you consider the virtualization of WrapPanel later?

https://github.com/WPFDevelopersOrg/WPFDevelopers.Minimal/blob/main/src/WPFDevelopers.Minimal/WPFDevelopers.Minimal.Shared/Controls/VirtualizingWrapPanel/VirtualizingWrapPanel.cs
